sdk/storage/inc/common/shared_request_options.hpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include "context.hpp"
+
+namespace Azure { namespace Storage {
+
+    struct SharedRequestOptions
+    {
+      /**
+       * @brief An optional operation timeout value in seconds. The period begins
+       *        when the request is received by the service. If the timeout value
+       *        elapses before the operation completes, the operation fails.
+      */
+      uint32_t Timeout = 0; // TODO: This is going to be changed into Nullable.
+
+      /**
+       * @brief Context for cancelling long running operations.
+      */
+      Azure::Core::Context Context;
+
+      /**
+       * @brief A UUID recorded in the analytics logs for troubleshooting
+                and correlation.
+      */
+      std::string ClientRequestId;
```

Comment:
client request id is added via pipeline policy

Reply:
Confirmed that using pipeline policy to set `ClientRequestId` is the correct approach. Removed in the latest update.